### PR TITLE
fix: fix focus indicator styling issue (vr-toolbar flag only) 

### PR DESCRIPTION
**Pull request by dpitcock**
### Description

fixing the focus indicatory styling on the visual refresh toolbar  where the indicator styling was being cur off by parent elements. 

Here the parent elements all have a height adjusted to 100% of the toolbar and the items aligned to the center, which allows for the indicator styling on the top and the bottom to be seen like such

Screenshot 2024-08-20 at 11 20 01

Also in this PR, the horizontal padding of the toolbar is set in two different elements. The left side remains in the `.universal-toolbar` with the right padding set as zero and the matching right padding is set in its child element with `.drawers-desktop-triggers-container, .drawers-mobile-triggers-container`. This allows for the right side styling of the focus indicator on the last item to show correctly

Related links, issue #, if available: n/a

This bug has been surfaced while working into the tooltips issue.

### How has this been tested?

The issue has been tested manually on firefox and chrome browser. A visual test of the focus inducator of the first and especially the last element should be added to ensure future changes do not have unintended changes to this focus indicator.

To test yourself, go to https://d21d5uik3ws71m.cloudfront.net/components/470e001b53e44b06d07239c29aa13c77d5bf255a/dev-pages/index.html#/light/app-layout/with-drawers?appLayoutWidget=true

Use the Tab to navigate the toolbar, left and right, and notice the indicator styling
Note: this does not address the focus control issue for opening the drawers

<details>
   <summary>Review checklist</summary>

_The following items are to be evaluated by the author(s) and the reviewer(s)._

#### Correctness

- _Changes include appropriate documentation updates._
- _Changes are backward-compatible if not indicated, see [`CONTRIBUTING.md`](https://github.com/cloudscape-design/components/blob/main/CONTRIBUTING.md#public-apis)._
- _Changes do not include unsupported browser features, see [`CONTRIBUTING.md`](https://github.com/cloudscape-design/components/blob/main/CONTRIBUTING.md#browsers-support)._
- _Changes were manually tested for accessibility, see [accessibility guidelines](https://cloudscape.design/foundation/core-principles/accessibility/)._

#### Security

- _If the code handles URLs: all URLs are validated through [the `checkSafeUrl` function](https://github.com/cloudscape-design/components/blob/main/src/internal/utils/check-safe-url.ts)._

#### Testing

- _Changes are covered with new/existing unit tests?_ existing
- _Changes are covered with new/existing integration tests?_ existing. 

</details>

---

By submitting this pull request, I confirm that you can use, modify, copy, and redistribute this contribution, under the terms of your choice.

### CODE REVIEW

File: src/app-layout/visual-refresh-toolbar/toolbar/styles.scss [hunk 1]
```diff
@@ -12,9 +12,10 @@
   border-block-end: 1px solid awsui.$color-border-divider-panel-bottom;
   box-sizing: border-box;
   padding-block: 0;
-  padding-inline: awsui.$space-m;
+  padding-inline: awsui.$space-m 0;
+  //right padding set in child trigger-container below for focus indicator to show correctly
   position: sticky;
-  z-index: 840;
```
Comment (georgylobko) [inline src/app-layout/visual-refresh-toolbar/toolbar/styles.scss:17]: I'm curious why did you remove this z-index?

File: src/app-layout/visual-refresh-toolbar/toolbar/styles.scss [hunk 2]
```diff
@@ -12,9 +12,10 @@
   border-block-end: 1px solid awsui.$color-border-divider-panel-bottom;
   box-sizing: border-box;
   padding-block: 0;
-  padding-inline: awsui.$space-m;
+  padding-inline: awsui.$space-m 0;
+  //right padding set in child trigger-container below for focus indicator to show correctly
```
Comment (georgylobko) [inline src/app-layout/visual-refresh-toolbar/toolbar/styles.scss:16]: this comment probably belongs to the line above, right?

File: src/app-layout/visual-refresh-toolbar/toolbar/styles.scss (hunk 1 above)
Comment (dpitcock) [reply src/app-layout/visual-refresh-toolbar/toolbar/styles.scss:17]: unintentional. Did a git reset and added back

File: src/app-layout/visual-refresh-toolbar/toolbar/styles.scss (hunk 2 above)
Comment (dpitcock) [reply src/app-layout/visual-refresh-toolbar/toolbar/styles.scss:16]: I set he padding on the right to 0 here so I is added in the child below

**Review by michaeldowseza — APPROVED** [review]

**Review by georgylobko — APPROVED** [review]
